packages/docusaurus-utils/src/gitUtils.ts

```diff
@@ -5,11 +5,15 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+/* eslint-disable max-classes-per-file */
```

Comment:
If that rule is on, you can turn it off.

Reply:
Ok